### PR TITLE
Handle inbound with EDD label

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: ndoh_hub/settings.py [hunk 1]
```diff
@@ -307,3 +307,4 @@
 if ENABLE_EVENTSTORE_WHATSAPP_ACTIONS:
     RAPIDPRO_OPERATOR_REPLY_FLOW = env.str("RAPIDPRO_OPERATOR_REPLY_FLOW")
     RAPIDPRO_UNSENT_EVENT_FLOW = env.str("RAPIDPRO_UNSENT_EVENT_FLOW")
+    RAPIDPRO_EDD_LABEL_FLOW = env.str("RAPIDPRO_EDD_LABEL_FLOW")
```
Comment (erikh360) [inline ndoh_hub/settings.py:310]: This env will need to be configured when you deploy.

**Review by DevChima — COMMENTED** [review]
Great

File: ndoh_hub/settings.py (hunk 1 above)
Comment (DevChima) [reply ndoh_hub/settings.py:310]: @erikh360 Where do I configure that variable. In the sandbox Turn webhook?

Comment (DevChima) [reply ndoh_hub/settings.py:310]: Seen. Thanks Erik

**Pull request merged**